wgsl/index.bs

```diff
@@ -1157,6 +1157,11 @@ access_mode
 
 ### Storable Types ### {#storable-types}
 
+A value placed in memory must be of a [=storable=] type.
+A storable type may have an explicit representation defined by WGSL,
+as described in [[#internal-value-layout]],
+or it may be opaque, such as for textures and samplers.
```

Comment:
Something reads wrong to me here. "A value placed in memory" is talking about stuff *in* memory, right? We aren't placing textures and samplers into memory, as far as I know. These are handles to something in memory. Maybe we can reword this?

Reply:
Right, thanks.  I was going for placed in a variable, but this is before variables have been introduced, and this was my attempt at avoiding circularity.  I'll try to reword it.